checker/build.xml

```diff
@@ -665,6 +665,13 @@
         </antcall>
     </target>
 
+    <target name="lock-safedefaults-tests" depends="jar,build-tests"
+            description="Run tests for the Lock Checker using -AuseSafeDefaultsForUnannotatedSourceCode">
```

Comment:
Minor: Instead of giving the parameter name, would the effect be helpful/easier to read?